tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "lib": [
+      "es2022" /* Allow for Object.entries, Error.cause */,
+      "dom",
+      "webworker.importscripts",
+      "scripthost"
+    ],
+    "jsx": "react",
```

```suggestion
    "jsx": "react",
```

Is this needed? this package doesn't contain any react code